community/README.md

```diff
@@ -49,4 +49,10 @@ We have weekly contributor meeting every Friday at 10am Indian Standard Time ove
 
 To add this event to your calendar, please send a mail to nisanta.sahoo@openebs.io or use the following [public event link](https://calendar.google.com/calendar/embed?src=openebs%40gmail.com&ctz=Asia%2FCalcutta).
 
-Map that to your local time with this [timezone table](https://www.google.com/search?q=1000+am+in+ist).
\ No newline at end of file
+Map that to your local time with this [timezone table](https://www.google.com/search?q=1000+am+in+ist).
+
+### Meetings
```

Comment:
heading might require a change -- "Meeting Resources" may be better or just "Links"

Reply:
@kmova Changed the title